assets/js/googlesitekit-adminbar.js

```diff
@@ -80,7 +85,11 @@ export class GoogleSitekitAdminbar extends Component {
 							mdc-layout-grid__cell--align-middle
 						">
 							<span className="googlesitekit-adminbar__subtitle">{ __( 'Stats for:', 'google-site-kit' ) }</span>
-							<h2 className="googlesitekit-adminbar__title">{ googlesitekit.pageTitle && decodeHtmlEntity( googlesitekit.pageTitle ) }</h2>
+							<h2 className="googlesitekit-adminbar__title">
+								{ '' !== pageTitle ?
```

Comment:
Maybe just to make sure, this could also be `pageTitle && pageTitle.length`?

Reply:
Why not just check for a truthy `pageTitle` since an empty string will be falsy?

Reply:
Adjusted in https://github.com/google/site-kit-wp/pull/543/commits/2b7baf77d072c7ac6b8d5cb5202f88d51868d246